keras_sklearn_compare_caip_e2e.ipynb

```diff
@@ -51,9 +36,14 @@
    },
    "outputs": [],
    "source": [
-    "# This is the Python package you'll use to access the What-if Tool (WIT)\n",
-    "# You can ignore the errors from this pip install\n",
-    "!pip install witwidget --quiet"
+    "# If you're running on Colab, you'll need to install the What-if Tool package and authenticate\n",
+    "import os\n",
+    "\n",
+    "if 'COLAB_GPU' in os.environ:\n",
+    "    !pip3 install witwidget --quiet\n",
```

Comment:
Is this line still correct if the user runs this colab with python2? I think pip should normally point to pip2 or pip3 depending on the active version of python, but not sure about this specific setup.

Reply:
Hmm, good point. If you're using a Python 3 notebook in CAIP, you need to `!pip3 install` to install packages. Any thoughts on how to handle? Should we just encourage folks to use Python 3 at the beginning of the notebook?

Reply:
I think we want these notebook to work in py2 or py3 both without any need for changes from users, as our other notebooks do.

can run "!python --version" and parse the output for 2.X or 3.X and set a var and then have if statement based on it for which pip to use.